spec/javascripts/support/jasmine_helper.rb

```diff
@@ -1,13 +1,10 @@
 #Use this file to set/override Jasmine configuration options
 #You can remove it if you don't need it.
 #This file is loaded *after* jasmine.yml is interpreted.
-#
-#Example: using a different boot file.
-#Jasmine.configure do |config|
-#   config.boot_dir = '/absolute/path/to/boot_dir'
-#   config.boot_files = lambda { ['/absolute/path/to/boot_dir/file.js'] }
-#end
-#
+
+Jasmine.configure do |config|
+  config.runner_browser = :chromeheadless
+end
```

Comment:
uses chromeheadless instead of phantomjs or selenium, thats the best alternative for jasmine afaict